promptsource/templates/math_qa/templates.yaml

```diff
@@ -30,25 +54,54 @@ templates:
 
       {{options}}|||
 
-      {{correct}}'
+      {{answer_choices[mapping[correct]]}}'
```

Comment:
you can use `{{correct}}` directly (also the mapping is missing "e" but since I suggest using `correct` directly, you don't need the mapping!